Frends.LDAP.SearchObjects/Frends.LDAP.SearchObjects/SearchObjects.cs

```diff
@@ -126,10 +123,45 @@ public static Result SearchObjects([PropertyTab] Input input, [PropertyTab] Conn
         {
             throw new Exception($"SearchObjects error: {ex}");
```

Comment:
While we are here lets improve this a bit as well to `throw new Exception("SearchObjects error", ex);`, so that the exception comes through as an inner exception